src/platform/forms/tests/forms.unit.spec.js

```diff
@@ -1,233 +1,304 @@
 import { expect } from 'chai';
-import fullSchema1010ez from 'applications/hca/config/form';
-import fullSchema0993 from 'applications/edu-benefits/0993/config/form';
-import fullSchema0994 from 'applications/edu-benefits/0994/config/form';
-import fullSchema0996 from 'applications/disability-benefits/996/config/form';
-import fullSchema1990 from 'applications/edu-benefits/1990/config/form';
-import fullSchema1990e from 'applications/edu-benefits/1990e/config/form';
-import fullSchema1990n from 'applications/edu-benefits/1990n/config/form';
-import fullSchema1995 from 'applications/edu-benefits/1995/config/form';
-import fullSchema5490 from 'applications/edu-benefits/5490/config/form';
-import fullSchema5495 from 'applications/edu-benefits/5495/config/form';
-import fullSchema527EZ from 'applications/pensions/config/form';
-import fullSchema526EZ from 'applications/disability-benefits/all-claims/config/form';
-import fullSchema530 from 'applications/burials/config/form';
-import fullSchema10007 from 'applications/pre-need/config/form';
-import fullSchema686 from 'applications/disability-benefits/686c-674/config/form';
-import fullSchemaFeedbackTool from 'applications/edu-benefits/feedback-tool/config/form';
-import fullSchema1010CG from 'applications/caregivers/config/form';
-import fullSchemaMDOT from 'applications/disability-benefits/2346/config/form';
-
 import { VA_FORM_IDS } from 'platform/forms/constants';
-
 import schemas from 'vets-json-schema/dist/schemas';
+import { externalServices } from 'platform/monitoring/DowntimeNotification';
+
+const path = require('path');
+const find = require('find');
+
+// Remap the formId to match the name in vets-json-schema
+const remapFormId = {
+  [VA_FORM_IDS.FORM_10_10EZ]: '10-10EZ',
+  [VA_FORM_IDS.FORM_21_526EZ]: '21-526EZ-ALLCLAIMS',
+};
 
-// Maps schema id to config id
-const mappedIds = [
-  '10-10EZ',
-  VA_FORM_IDS.FORM_21_526EZ,
-  VA_FORM_IDS.FORM_21_686C,
-  VA_FORM_IDS.FORM_21P_527EZ,
-  VA_FORM_IDS.FORM_21P_530,
-  VA_FORM_IDS.FORM_22_0993,
-  VA_FORM_IDS.FORM_22_0994,
-  VA_FORM_IDS.FORM_20_0996,
-  VA_FORM_IDS.FORM_22_1990,
-  VA_FORM_IDS.FORM_22_1990E,
-  VA_FORM_IDS.FORM_22_1990N,
-  VA_FORM_IDS.FORM_22_1995,
-  VA_FORM_IDS.FORM_22_5490,
-  VA_FORM_IDS.FORM_22_5495,
-  VA_FORM_IDS.FORM_40_10007,
-  VA_FORM_IDS.FEEDBACK_TOOL,
-  VA_FORM_IDS.FORM_10_10CG,
-  VA_FORM_IDS.FORM_VA_2346A, // MDOT
+// These form IDs have a config/form.js file but the formId is not found in vets-json-schema/dist/schemas
+const missingFromVetsJsonSchema = [
+  VA_FORM_IDS.FORM_HC_QSTNR,
+  VA_FORM_IDS.FORM_COVID_RESEARCH_VOLUNTEER,
```

Comment:
With change to `constants` this line should be removed

Reply:
removed